documentation/release-notes/topics/version_3.rst

```diff
@@ -3,6 +3,107 @@
+
+- Power on
+- Power off
+- Power cycle
+
+Supported Rack PDU hardware.
+
+.. note:: Please contact our Sales team if you are interested in getting an Outlet Control license.
+
+.. warning:: The latest Data Collector software is needed to take advantage of this feature.
+
+========================================
+Enhanced Licensed Feature: Rack Security
+========================================
+Rack Security control actions have been enhanced to integrate with Work Orders. A rack open operation will create a corresponding Work Order to document and track the action.
+
+.. note:: Please contact our Sales team if you are interested in getting a Rack Security license.
+
+.. warning:: The latest Data Collector software is needed to take advantage of this feature.
+
+===============================================
+Enhanced Licensed Feature: ServiceNow CMDB Sync
+===============================================
+Administrators are now able to test the ServiceNow credentials from the configuration page before any sync job is attempted.
+
+.. note:: Please contact our Sales team if you are interested in getting a ServiceNow CMDB Integration license.
+
+============================================
+New Feature: Automated Catalog Data Feedback
+============================================
+A new feature was added to enable automated feedback to Hyperview when previously unknown asset models are added. This feature will allow Hyperview to add support to new models more rapidly.
+
+Hyperview Administrators can manage this setting from the "Settings -> General -> Catalog Data" page.
+
+It is turned *Off* by default.
+
+=============================
+New Feature: Sensor Dashboard
+=============================
+A new sensor dashboard has been added to allow users to better visualize collected sensor data for an asset. Users will be able to toggle between list and card view for sensors.
+
+===============================
+Application Layout Improvements
+===============================
+- The search navigation bar has been added to the top of the product to allow for better access to Quick and Advanced Search features
+- The Account Management link has been moved from the bottom left of the application layout to the top right
+
+================================================
+2D and 3D Layout: Enhanced Heatmap Visualization
+================================================
+Heatmap visualization has been enhanced to be more accurate and more performant.
+
+In addition, there was a host of bug fixes, usability and performance improvements.
+
+=================================
+Sensor Data Download Improvements
+=================================
+Sensor data export has been improved to allow for up to 180 days of raw or daily summary data. Provided that the range falls within license limits and there is corresponding collected data for the selected sensor. This feature is available from both the application GUI and the API.
```

Comment:
Sensor data export has been improved to allow for up to 180 days of raw or daily summary data (provided that the range falls within license limits and corresponding data exists).